### ISSUE TITLE
PseudoElement fails to find new content during recalc (ie7-recalc.js)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#122]
```
What steps will reproduce the problem?
1. Using both the IE7/IE8 script and the IE7-recalc script
2. After document.readyState == "complete", add a element with content 
"PseudoElement" style.
3. Run document.recalc();

What is the expected output? What do you see instead?
The content should be applied to the new element. Instead it only applies 
styles to PseudoElements found when script first loaded.

What version of the product are you using? On what operating system?
IE8.js 2.0b3; Windows XP; IE6 & IE7;

Please provide any additional information below.
Fix: Re-evaluate PseudoElement matches.
IE7.PseudoElement.prototype.extend({
  recalc: function() {
    if (!this.content) return ;
    this.init(); // re-evaluate "this.match" array;
    this.base();
  },
  create: ...
});



```

Original issue reported on code.google.com by `dark.acc...@gmail.com` on 29 Oct 2008 at 10:05

**Comment by GoogleCodeExporter** [comment]
```
I second on this. Just including ie7-recalc.js in my page will fail to load the
before-content AT ALL.
Removing the include will display the content just fine, but I'm working with an
AJAX-Tree that needs recalc() after each click in case new branches are 
displayed and
need rendering of the new content.
(For anyone interested: I'm trying to get Ajax.Tree by Colin Mollenhour to work 
with
IE, the span.mark-elements drive me nuts.)

B.
```

Original comment by `buegelfa...@gmail.com` on 2 Jul 2009 at 3:05

**Comment by GoogleCodeExporter** [comment]
```
I'm not currently supporting ie7-recalc but I will look into it later.
```

Original comment by `dean.edw...@gmail.com` on 12 Feb 2010 at 2:31
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `dean.edw...@gmail.com` on 14 Feb 2010 at 10:24
- Changed title: **PseudoElement fails to find new content during recalc (ie7-recalc.js)**

**Comment by GoogleCodeExporter** [comment]
Original comment by `dean.edw...@gmail.com` on 14 Feb 2010 at 10:24
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**
